Automattic/wp-calypso
# Activity log: Add rewind toggle

Adds rewind toggle. Currently the toggle is always in "deactivated," "NotToggling" state. That will need to be corrected selecters land in #14900 ✅ 

Includes #14976 which makes the toggles behave correctly and update after toggling.

To test:
1. Vist one
   * https://calypso.live/stats/activity?branch=add/activitylog-rewind-toggle
   * http://calypso.localhost:3000/stats/activity
1. Pick a Jetpack site
1. The toggle should be disabled until sites are loaded (requires `siteId`).
1. Click the toggle.
1. Ensure you see an activate request fire in network.
1. Try toggling on and off.

~Note: `REWIND_ACTIVATE_SUCCESS` or `REWIND_DEACTIVATE_SUCCESS` is dispatched when the request comes back, reducer should be updated to correctly adjust state.~ Implemented in included #14976

All toggles states:

![toggles](https://user-images.githubusercontent.com/841763/26975285-6afe90a6-4d1f-11e7-870b-c9db57166118.gif)

Fixes #14676.
